### PR TITLE
temporary bitcoin node

### PR DESCRIPTION
**Pull request by tupakkatapa**
Opening a place for discussion about a temporary Bitcoin node; should not be merged.

### CODE REVIEW

**Comment by tupakkatapa** [comment]
@jhvst Does this have the necessary bits and pieces? What files and directories do I have to manually create in addition to the SSH private key?

File: nixosConfigurations/adhoc-ephemeral-alpha/default.nix [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+    description = "electrum rpc";
+    requires = ["wg-quick-wg0.service" "bitcoind-mainnet.service"];
+    after = ["wg-quick-wg0.service" "bitcoind-mainnet.service"];
+
+    script = ''      ${pkgs.electrs}/bin/electrs \
+            --db-dir /mnt/bitcoin/electrs/db \
+            --cookie-file /mnt/bitcoin/bitcoind/.cookie \
+            --network bitcoin \
+            --electrum-rpc-addr 192.168.100.10:50001
```
Comment (jhvst) [inline nixosConfigurations/adhoc-ephemeral-alpha/default.nix:76]: this has to be host wireguard addr

File: nixosConfigurations/adhoc-ephemeral-alpha/default.nix [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+  ...
+}: let
+  # General
+  infra.ip = "192.168.100.121";
+  sshKeysPath = "/mnt/bitcoin/id_ed25519";
+in {
+  homestakeros = {
+    # Localization options
+    localization = {
+      hostname = "adhoc-ephemeral-alpha";
```
Comment (jhvst) [inline nixosConfigurations/adhoc-ephemeral-alpha/default.nix:15]: i think we have to come up with a new zone name, feel free to choose

**Review by jhvst — COMMENTED** [review]
otherwise good

**Comment by jhvst** [comment]
closed as redundant

**Pull request closed without merging**